pkgs/applications/misc/ranger/default.nix

```diff
@@ -51,6 +51,6 @@ python3Packages.buildPythonApplication rec {
     homepage = http://ranger.github.io/;
     license = licenses.gpl3;
     platforms = platforms.unix;
-    maintainers = [ maintainers.magnetophon ];
+    maintainers = [ maintainers.toonn maintainers.magnetophon ];
```

Comment:
This must fit the attribute name in `maintainer-list.nix`, so either you write `maintainers.toonnolten` here or you change it to `toonn = ..` in `maintainer-list.nix`. I recommend the second option because it is more convenient to have your maintainer name equal to your github handle.

Reply:
Woops, that makes a heap of sense. I was mimicking the other entries while editing the list.